src/qibolab/platform.py

```diff
@@ -462,15 +412,14 @@ def create_CNOT_pulse_sequence(self, qubits):
 
     def create_MZ_pulse(self, qubit):
         qubit = self.get_qubit(qubit)
-        return replace(qubit.native_gates.MZ, channel=qubit.readout.name)
+        return replace(qubit.native_gates.MZ)
```

Comment:
At this point, you don't need to replace any longer (at most to copy, but being immutable, it should not be required)
```suggestion
        return qubit.native_gates.MZ
```

Reply:
`create*` methods of platform no longer exist